test/test_translator.py

```diff
@@ -138,8 +136,73 @@ def test_parse_polyline() :
         ]
     assert  test_coordinates == valid_coordinates
 
+def test_parse_polyline_empty_string() :
+
+    test_polyline= None
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    valid_coordinates= []
+    assert  test_coordinates == valid_coordinates
+
+def test_parse_polyline_invalid_data() :
+    test_polyline= 'invalid' 
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    valid_coordinates= []
+    assert  test_coordinates == valid_coordinates
 
-def test_get_road_direction():
+def test_parse_polyline_invalid_coordinates() :
+    try:
+        test_polyline= 'a,b,c,d' 
+        test_coordinates=icone_translator.parse_polyline(test_polyline)
+        assert False
+    except RuntimeError: 
+        assert True
+
+def test_validate_incident_valid_data():
+  test_valid_output = {
+    '@id': 'U13631595_202012160845',
+    'updatetime': '2020-12-16T17:18:00Z',
+    'starttime': '2020-12-07T14:18:00Z',
+    'description': 'Road constructions are going on', 
+    'creationtime': '2020-12-13T14:18:00Z', 
+    'location': {
+      'polyline': '34.8380671,-114.1450650,34.8380671,-114.1450650',
+      'street': 'I-70 N'
+    }
+  }
+  assert icone_translator.validate_incident(test_valid_output) == True
+
+def test_validate_incident_missing_required_field():
```

Comment:
Which field are we testing here that is missing? This should be included in the name so we don't have to go looking.

Reply:
i renamed it to def test_validate_incident_missing_required_field_description():

since in that test case description was missing